flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlTableSourceFactory.java

```diff
@@ -244,6 +246,34 @@ private static void validateSpecificOffset(ReadableConfig config) {
         }
     }
 
+    private static void validateTimeStamp(ReadableConfig config) {
+        Optional<Long> timeStamp = config.getOptional(SCAN_STARTUP_TIMESTAMP_MILLIS);
+        if (!timeStamp.isPresent()) {
+            throw new ValidationException(
+                    String.format(
+                            "Unable to find a valid timestamp. %s are required.",
```

Comment:
String.format("Option %s are required when setting the option  %s to %s.", SCAN_STARTUP_TIMESTAMP_MILLIS.key(), SCAN_STARTUP_MODE.key(), SCAN_STARTUP_MODE_VALUE_TIMESTAMP)